rocAL/rocAL/source/augmentations/geometry_augmentations/node_resize_mirror_normalize.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include <vx_ext_rpp.h>
+#include <graph.h>
+#include <cmath>
+#include "node_resize_mirror_normalize.h"
+#include "exception.h"
+
+ResizeMirrorNormalizeNode::ResizeMirrorNormalizeNode(const std::vector<Image *> &inputs, const std::vector<Image *> &outputs) :
+        Node(inputs, outputs), _mirror(MIRROR_RANGE[0], MIRROR_RANGE[1])
+{
+}
+
+void ResizeMirrorNormalizeNode::create_node()
+{
+    if(_node)
+        return;
+
+    std::vector<vx_float32> mean_vx, std_dev_vx;
+    _dest_width_val.resize(_batch_size);
+    _dest_height_val.resize(_batch_size);
+    mean_vx.resize(_batch_size * 3);
+    std_dev_vx.resize(_batch_size * 3);
+    for (uint i = 0; i < _batch_size; i++) {
+        mean_vx[3 * i] = _mean[0];
+        mean_vx[3 * i + 1] = _mean[1];
+        mean_vx[3 * i + 2] = _mean[2];
+
+        std_dev_vx[3 * i] = _std_dev[0];
+        std_dev_vx[3 * i + 1] = _std_dev[1];
+        std_dev_vx[3 * i + 2] = _std_dev[2];
+    }
```

Comment:
Can we change this to mem_copy?

Reply:
Mean_vx and std_dev_vx are vectors of size BS*3 while _mean and _std_dev are vectors of size 3. If we opt for memcpy it still needs to done over a loop so this seems to be the clean option